lib/commands/gesture.js

```diff
@@ -59,13 +59,19 @@ commands.nativeClick = async function (el) {
 };
 
 function isDrag (gestures) {
-  return (
-      gestures.length === 4 &&
+  if (gestures.length === 4 &&
       gestures[0].action === 'press' &&
       gestures[1].action === 'wait' &&
       gestures[2].action === 'moveTo' &&
-      gestures[3].action === 'release'
-  );
+      gestures[3].action === 'release') {
+    return true;
+  } else if (gestures.length === 3 &&
```

Comment:
still seems wrong to me that something without any moveTo can be a drag. are we sure we don't want to just fix the clients?